local/local.go

```diff
@@ -14,6 +14,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/docker/docker/pkg/jsonmessage"
+
```

Comment:
can we group this with the other external imports below?

Reply:
Done!